src/SigSpec.AspNetCore/SigSpecDocumentGeneratorSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using SigSpec.Core;
+
+namespace SigSpec.AspNetCore.Middlewares
+{
+    public class SigSpecDocumentGeneratorSettings : SigSpecGeneratorSettings
+    {
+        public string DocumentName { get; set; } = "v1";
```

Comment:
Is it possible to have versioned SignalR hubs?

Reply:
I’d say it’s less about versions but a group of “operations” and about being able to register multiple docs with eg a different list of hubs

Reply:
So an actual versioning is up to the implementor, eg for v2 you’d have a v2 hub and only register this for the v2 document

Reply:
Ok, yep that makes sense